src/sentry/replays/consumers/recording_buffered.py

```diff
@@ -238,6 +239,23 @@ def process_message(buffer: RecordingBuffer, message: bytes) -> None:
             decoded_message["retention_days"],
             parsed_recording_data,
         )
+
+        if replay_actions is not None:
+            buffer.replay_action_events.append(replay_actions)
```

Comment:
do we need to move the message to the DLQ somehow? `catch` seems to only log it

Reply:
Only if the message is retry-able after programmer intervention.  So something like a service outage isn't a good fit for a DLQ.  We intentionally crash loop in those scenarios and wait for the service to become available.

We wouldn't want to DLQ the message that caused INC-630 because we have no capability of parsing it.  It should be silently dropped (we log for tracking but otherwise don't care).  The replay message is still partially processed.  The rage clicks are not sent but we still have the core functionality of the message processor working.